### PR TITLE
feat: update user status component

### LINKED ISSUE #48: Update user status card
Blocked by #45 

### **Objective:**
Update the user status card
### **Description:**

* Update the userStatus card component `/components/cards/UserStatus.tsx`
* The component has a state to save whether the status is visible to any friends or not
* When the status is visible, it should show the visible friends, and when it's not,  show:`status in not visible to any...` according to the Figma design

### **Acceptance Criteria:**
* the component is updated with the styles according to the Figma styles

### **Design:**
[Links or screenshots of the design]

Image

### PR DESCRIPTION
**Pull request by mhenriette**
Closes #48

### CODE REVIEW

File: components/cards/UserStatus.tsx [hunk 1]
```diff
@@ -4,14 +4,13 @@ import { theme } from "@/theme"
...
 import { HomeScreenProps } from "@/screens/Home"
-import UserInfo from "@/components/UserInfo"
-import EllipsisIcon from "@/components/vectors/EllipsisIcon"
 import Animated, {
   interpolate,
   SharedValue,
   useAnimatedStyle,
   withTiming,
 } from "react-native-reanimated"
+import UserAvailability from "./UserAvailability"
```
Comment (igorntwari) [inline components/cards/UserStatus.tsx:13]: use dynamic render here

**Review by igorntwari — CHANGES_REQUESTED** [review]

File: components/cards/UserStatus.tsx (hunk 1 above)
Comment (mhenriette) [reply components/cards/UserStatus.tsx:13]: Why is it needed?

**Review by Odile078 — APPROVED** [review]

File: components/cards/UserStatus.tsx (hunk 1 above)
Comment (igorntwari) [reply components/cards/UserStatus.tsx:13]: i meant to use alias import

File: components/cards/UserStatus.tsx [hunk 2]
```diff
@@ -61,24 +59,46 @@ export default function UserStatus({
...
+              </CustomText>
+              {visibleFriendsList && (
+                <CustomText
+                  size="sm"
+                  fontFamily="writer-mono"
+                  style={styles.statusText}>
+                  {visibleFriendsList}.
+                </CustomText>
+              )}
+            </>
```
Comment (serapieTuyishime) [inline components/cards/UserStatus.tsx:101]:
```suggestion
          <>
            <CustomText
              size="sm"
              fontFamily="writer-mono"
              style={styles.statusText}>
              {friends.length
                ? `Visible to ${friends.length} friends`
                : "This status is not visible to anyone"}
            </CustomText>
            <CustomText
              size="sm"
              fontFamily="writer-mono"
              style={styles.statusText}>
              {visibleFriendsList
                ? `${visibleFriendsList}.`
                : "Tap to edit your preferences."}
            </CustomText>
          </>
```

**Review by serapieTuyishime — APPROVED** [review]

**Review by igorntwari — APPROVED** [review]

**Review by serapieTuyishime — APPROVED** [review]

**Review by AnasAlamin — APPROVED** [review]

**Pull request merged**